app/views/tag/blog2.html.erb

```diff
@@ -13,17 +36,17 @@
     <p>
     Here is the lorem ipsum content
     <br>
-    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod 
-      tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim 
-      veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex 
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
```

Comment:
Hi @gautamig54 
Damn!! The UI looks great.
Regarding spaces ? We are going to remove this lorem ipsum soon.
Is it necessary ?
:thinking:

Reply:
Thanks @CleverFool77 !!!
I think the spacing happened unintentionally. I'll resolve it.